packages/manager/apps/dedicated/client/app/account/user/users/sso/delete/user-users-sso-delete.html

```diff
@@ -0,0 +1,15 @@
+<div data-ng-controller="UserAccountUsersSSODeleteCtrl as $ctrl">
+    <div
+        data-wizard
+        data-wizard-on-cancel="resetAction"
+        data-wizard-on-finish="deleteSSO"
+        data-wizard-title="::'user_users_sso_delete_modal_title' | translate"
+    >
+        <div data-wizard-step>
+            <p
+                class="text-danger"
+                data-translate="user_users_sso_delete_modal_step1_question"
+            ></p>
+        </div>
+    </div>
+</div>
```

Comment:
the `ng-controller` is useless:
```suggestion
    <div
        data-wizard
        data-wizard-on-cancel="resetAction"
        data-wizard-on-finish="deleteSSO"
        data-wizard-title="::'user_users_sso_delete_modal_title' | translate"
    >
        <div data-wizard-step>
            <p
                class="text-danger"
                data-translate="user_users_sso_delete_modal_step1_question"
            ></p>
        </div>
    </div>
```